### DISCUSSION TITLE
Suggestion: Search for Replies on my posts (Included, or alternatives)

### DISCUSSION THREAD

**Discussion by ledgeri** [ThioJoe/YT-Spammer-Purge#313] [Ideas]
Around 2-3 months ago i have started reciving notifications about my comments getting replies, but all clearly was bot comment.
It was annoying because for 2 weeks, or a month or so i daily recived nasty invitations, you know the kind... 

Currently the program can take care of Channels (scanning for comments on my channel, or for others') or videos, but with this "scan who is replied to my comments" would clear "the picture of my comments", if you get the idea... (i know, others' channels' comments are just reportable, but still) I can find my or others' channel, but i do not really can find all of my comments, and if i clearied, or want to clear my emails, if the notifications even turned on for those. So tracking back all the comments is only possible thrugh the yt-API, and this program went soo deep in this regard, this addition would be a nice one. 
Writing this i just realised, "listing all the channels, where i commented" could provide similar results. with the current state of the program...

Anyways, thanks ThioJoe

**Comment by ledgeri** [comment]
(if you know how to force the api to drop "channels where i ever commented", that would solve the issue too

**Reply by Kurdakov** [reply]
there is no API feature, but it is possible to find most of your comments via https://www.alphr.com/how-to-view-your-youtube-comment-history/

**Reply by ledgeri** [reply]
> there is no API feature, but it is possible to find most of your comments via https://www.alphr.com/how-to-view-your-youtube-comment-history/

Yeah, but that is a constantly loading endless, full youtube page, not a snipy-snappy minimaly texted program, like the main topic's is... (forgot about it though, thx for the reminder :) )

**Comment by KendallDoesCoding** [comment]
> Around 2-3 months ago i have started reciving notifications about my comments getting replies, but all clearly was bot comment. It was annoying because for 2 weeks, or a month or so i daily recived nasty invitations, you know the kind...
> 
> Currently the program can take care of Channels (scanning for comments on my channel, or for others') or videos, but with this "scan who is replied to my comments" would clear "the picture of my comments", if you get the idea... (i know, others' channels' comments are just reportable, but still) I can find my or others' channel, but i do not really can find all of my comments, and if i clearied, or want to clear my emails, if the notifications even turned on for those. So tracking back all the comments is only possible thrugh the yt-API, and this program went soo deep in this regard, this addition would be a nice one. Writing this i just realised, "listing all the channels, where i commented" could provide similar results. with the current state of the program...
> 
> Anyways, thanks ThioJoe

Same, but theres a easy fix to this, just mute the channel from showing up in your notifications? 

How to do this?
1. Go to notifications tap
2. Hover hover notification
3. Go to notification settings
4. Mute channel from displaying in notifcs. Then its harmless

**Reply by ledgeri** [reply]
Sorry for the late reply:

Sure, i thought about that, but all was an other channel 95% with the same content...
2 possibilities: everyone got this under his/her video, and i recived a good bunch randomly, OR the bot hooked onto me for a good month... IDK
